Gems/Atom/Feature/Common/Assets/Materials/Types/MaterialInputs/GeneralCommonPropertyGroup.json

```diff
@@ -14,7 +14,7 @@
             "displayName": "Apply Specular AA",
             "description": "Whether to apply specular anti-aliasing in the shader.",
             "type": "Bool",
-            "defaultValue": false,
+            "defaultValue": true,
```

Comment:
Not every material type uses this file, some have inlined the same content. Search for "Apply Specular AA" to see those other locations.